multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -48,7 +48,7 @@
 @SuppressWarnings({"unchecked", "rawtypes"}) // core’s fault
 public class WorkflowMultiBranchProject extends MultiBranchProject<WorkflowJob,WorkflowRun> {
 
-    static final String SCRIPT = "jenkins.groovy";
+    static final String SCRIPT = "Jenkinsfile";
```

Comment:
My preference is that the file name should be configurable by the user but **default** to `Jenkinsfile`. The file is acting as a marker file to identify the branches that should be built. It is rather arrogant to assume that everyone will want to use `Jenkinsfile` as the file name.

OTOH adding the file name being configurable at a future date is not prevented... just more I am stating my opinion - influenced by experience with literate - given that we are touching this behaviour here